cmd/server/main.go

```diff
@@ -21,23 +22,35 @@ var (
 
 func main() {
 	logging.SetDebugLogging()
-	logging.SetLogLevel("*", "error")
+	logging.SetLogLevel("*", "info")
+	logging.SetLogLevel("rpc", "error")
+	logging.SetLogLevel("index*", "debug")
 	instrumentationSetup()
 
+	// ToDo: Flags for configuration
+
 	lotusAddr, token := tests.ClientConfigMA()
 	grpcAddr, err := ma.NewMultiaddr(grpcHostAddr)
 	if err != nil {
-		panic(err)
+		log.Errorf("invalid grpc host addr: %s", err)
+		os.Exit(-1)
+	}
+	repoPath, err := os.UserHomeDir()
+	if err != nil {
+		log.Errorf("error getting home dir: %s", err)
+		os.Exit(-1)
 	}
 	conf := server.Config{
 		LotusAddress:    lotusAddr,
 		LotusAuthToken:  token,
 		GrpcHostAddress: grpcAddr,
+		RepoPath:        filepath.Join(repoPath, ".texfc"),
```

Comment:
For the moment the default repo path is `~/.texfc`, kind of arbitrary. As ToDo mentions, latter could be flagged.